src/animation/interpolants/SlerpInterpolant.js

```diff
@@ -0,0 +1,40 @@
+/**
+ *
+ * Spherical linear quaternion interpolant.
+ *
+ *
+ * @author tschw
+ */
+
+THREE.SlerpInterpolant = function( times, values, stride, result ) {
```

Comment:
You could call this QuaternionLinearInterpolatant.  Makes it clearer that it is akin to LinearInterpolatant.  And then we can introduce a QuaternionCubicInterpolant/QuaternionSmoothInterpolant as the analogy to SmoothInterpolatant.